### ISSUE TITLE
Empty initial inclusion criteria result in nonfunctional SQL code

### ISSUE THREAD

**Issue by Spayralbe** [OHDSI/Atlas#270]
When a cohort's initial event inclusion criteria are present, but without any criteria provided (see picture), the generated SQL code will not work.

![qlw](https://cloud.githubusercontent.com/assets/14215796/19609496/d6d3c6b4-97d7-11e6-8a74-9731f10e57cd.png)

PostgreSQL example of the code that gives the problem:

```
CREATE TEMP TABLE qualified_events

AS
SELECT
 event_id, person_id, start_date, end_date, op_start_date, op_end_date

FROM

(
  select pe.event_id, pe.person_id, pe.start_date, pe.end_date, pe.op_start_date, pe.op_end_date, row_number() over (partition by pe.person_id order by pe.start_date ASC) as ordinal
  FROM primary_events pe

JOIN (
select 0 as index_id, event_id
FROM
(
  select event_id FROM
  (

  ) CQ
  GROUP BY event_id
  HAVING COUNT(index_id) = 0
) G
) AC on AC.event_id = pe.event_id

) QE

;
```

**Comment by chrisknoll** [comment]
This is true, I think what's required here is that when handling a 'criteria group' if there's nothing in CriteriaList or CriteriaGroup or Demographics, then there's nothign to do here , and don't try to create the inclusion criteria.

**Comment by chrisknoll** [comment]
This is fixed with PR OHDSI/webapi#156
